pipeline_dp/aggregate_params.py

```diff
@@ -129,21 +129,20 @@ class CountParams:
             unit of privacy (e.g., a user) can participate.
         max_contributions_per_partition: Bounds the number of times one unit of
             privacy (e.g. a user) can contribute to a partition.
+        partition_extractor: A function for partition id extraction from an RDD record.
```

Comment:
please RDD record -> record or "collection record"
 everywhere in this file

Explanation: RDD is a Spark term, this class was introduced initially for Spark only and than used for Beam, but the comment stale

Reply:
Thanks! fixed.